### PR TITLE
Release v48.20.0

### PR DESCRIPTION
**Pull request by jag-sfdc**
### What does this PR do?
Cherry picked commits from develop branch to be merged to master.

### What issues does this PR fix or reference?
N/A

### CODE REVIEW

File: packages/salesforcedx-vscode-core/src/messages/i18n.ts [hunk 1]
```diff
@@ -52,6 +52,8 @@ export const messages = {
   parameter_gatherer_enter_file_name: 'Enter desired filename',
   parameter_gatherer_enter_dir_name:
     'Enter desired directory (Press Enter to confirm or Esc to cancel)',
+  parameter_gatherer_enter_lwc_name:
+    'Enter desired Lightning Web Component (Press Enter to confirm or Esc to cancel)',
```
[the remark this answers is not in the document]
Comment (jag-sfdc) [reply packages/salesforcedx-vscode-core/src/messages/i18n.ts:56]: Taken care of the other issues. I'm not sure about this one. Lines 54 and 56 both use Esc and don't find "Escape" usage in other i18n files I looked.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Cancelling a command execution should not throw errors ([PR #2232](https://github.com/forcedotcom/salesforcedx-vscode/pull/2232))
+
+#### salesforcedx-vscode-lwc
+
+- -`SFDX: Start Local Development Server` and `SFDX: Preview Component Locally` use the new url path and correct port ([PR #2239](https://github.com/forcedotcom/salesforcedx-vscode/pull/2239))
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md:11]:
```suggestion
-`SFDX: Start Local Development Server` and `SFDX: Preview Component Locally` use the new url path and correct port ([PR #2239](https://github.com/forcedotcom/salesforcedx-vscode/pull/2239))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Add `SFDX: Install Package` command ([PR #2244](https://github.com/forcedotcom/salesforcedx-vscode/pull/2244), [Issue #2140](https://github.com/forcedotcom/salesforcedx-vscode/issues/2140)) - Contribution by [gabrielbarker](https://github.com/gabrielbarker)
+
+- - Add `SFDX: Create Lightning Web Component Test` command ([PR #2228](https://github.com/forcedotcom/salesforcedx-vscode/pull/2228))
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md:19]:
```suggestion
- Add `SFDX: Create Lightning Web Component Test` command ([PR #2228](https://github.com/forcedotcom/salesforcedx-vscode/pull/2228))
```

File: packages/salesforcedx-test-utils-vscode/package.json [hunk 4]
```diff
@@ -2,14 +2,14 @@
   "name": "@salesforce/salesforcedx-test-utils-vscode",
   "displayName": "SFDX Test Utilities for VS Code",
   "description": "Provides test utilies to interface the SFDX libraries with VS Code",
```
[the remark this answers is not in the document]
Comment (sfsholden) [reply packages/salesforcedx-test-utils-vscode/package.json]: @ekapner Great catches.

File: packages/salesforcedx-vscode-core/src/messages/i18n.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ekapner) [reply packages/salesforcedx-vscode-core/src/messages/i18n.ts:56]: Ok, thanks for checking

**Pull request merged**